ffcx/codegeneration/geometry.py

```diff
@@ -121,7 +121,7 @@ def facet_reference_edge_vectors(tablename, cellname):
     return L.ArrayDecl(symbol, values=out, const=True)
 
 
-def reference_facet_normals(tablename, cellname):
+def reference_normals(tablename, cellname):
```

Comment:
This would be `facet_outward_normals`.

Reply:
Then we end up with a mixture of `basix` or `ufl`-style nomenclature.
Isn't it better to stick with UFL all the way through (as basix doesn't have concepts such as reference_facet_edge_vectors).

Reply:
Ah, I see, I wrongly thought you went for basix naming. All UFL-like works too 👍 .